clap_derive/src/derives/arg_enum.rs

```diff
@@ -47,9 +47,8 @@ pub fn gen_for_enum(name: &Ident, attrs: &[Attribute], e: &DataEnum) -> TokenStr
     );
 
     let lits = lits(&e.variants, &attrs);
-    let variants = gen_variants(&lits);
-    let from_str = gen_from_str(&lits);
-    let as_arg = gen_as_arg(&lits);
+    let arg_values = gen_arg_values(&lits);
+    let arg_value = gen_arg_value(&lits);
```

Comment:
```suggestion
    let as_arg_value = gen_as_arg_value(&lits);
```

Once the trait method is renamed